al2/graalvm/11/gradle/{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM public.ecr.aws/amazonlinux/amazonlinux:2
+{%- if cookiecutter.architectures.value|length == 1 and cookiecutter.architectures.value[0] == 'arm64' %}
```

Comment:
Will customer still hit the same issue if they specify multiple archs?
Or do I misunderstand something? It seems `--architecture` in `sam init` only accepts one value... is it just the cookiecutter `architectures` was designed in a way to accept multiple values?

Reply:
yes, `sam init` accepts only one architecture, but in cookiecutter we set the architectures attribute to be a list. I did not want to change its type, I kept as a list with assumption that it will always contain one value.

Reply:
as we discussed in chat. we could not support multi architectures for this template as GraalVM does not support multi architectures. 
Also, changing `architectures` attribute in cookiecutter requires to do a change in sam cli to set it to a string in specific cases.
So we can go on with this solution as customers could not use sam init to set multiple architectures while initializing this template